source/Core/Inc/Settings.h

```diff
@@ -110,6 +110,12 @@ typedef enum {
   NO_DYNAMIC = 0, // PPS + EPR disabled, fixed PDO only
 } usbpdMode_t;
 
+typedef enum {
+  DISABLED = 0, // Locking buttons is disabled
+  BOOST    = 1, // Locking buttons for Boost mode only
+  FULL     = 2, // Locking buttons fully
```

Comment:
```suggestion
  FULL     = 2, // Locking all buttons
```

Reply:
Are you sure that this will be better? I just tried to keep the code as coherent as possible in accordance with the other parts including [description itself](https://github.com/Ralim/IronOS/blob/84e53e11e0835589d88fd4c231cc68883fe74fff/Translations/translation_EN.json#L158) and [documentation](https://github.com/Ralim/IronOS/blob/84e53e11e0835589d88fd4c231cc68883fe74fff/Documentation/Settings.md?plain=1#L249) to avoid any unnecessary ambiguity and not to puzzle anyone without extra need in the future.

We could change all the other descriptions after the suggestion though, but it would lead to the need of updating all translation files as well, which is - in my very humble opinion - total overkill.

TL;DR: this option-related value is notated as `full` in _the description_ and in _the documentation_ so why it should be `all` in _the code_ exactly? Just curious.

Reply:
It can be full, mostly just wanted to not have "Locking buttons fully" as it feels a very clunky description, and we are not length limited here like we are in the description.

Reply:
Aha, ok. I think I got it... Let me prepare a slightly updated solution then.